### DISCUSSION TITLE
Is it right that tts.edge action not showing add entity buttons?

### DISCUSSION THREAD

**Discussion by canedje** [zachowj/node-red-contrib-home-assistant-websocket#1643] [General]
Is it right that tts.edge action not showing add entity buttons? And you need to add entity at Data field?
![image](https://github.com/user-attachments/assets/aba17caf-17cd-452f-b096-5ded0b7ab4ee)
As it is working with media_player?
![image](https://github.com/user-attachments/assets/7805de3f-3c13-4789-a0a9-ae0abdab81cd)

**Comment by zachowj** [comment]
Looking at the documentation for `edge_say`, the `entity_id` should be placed under the data field, not the targets field. That's why the targets don't show up when you have `edge_say` selected as the action.

Refer to the example here:  
<https://github.com/hasscc/hass-edge-tts?tab=readme-ov-file#basic-example>

**Comment by canedje** [comment]
Thanks for the answer. Strange there is different approach
